lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc

```diff
@@ -32,6 +32,15 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
       'maxlength' => '140',
     ),
   ];
+  $form['custom_social_sharing']['social_share_custom_description'] = [
+    '#type' => 'textfield',
```

Comment:
I feel like this should be a `textarea` so it can be longer, seems like a place someone might want some more copy potentially